src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -82,6 +82,14 @@ protected function describeRouteCollection(RouteCollection $routes, array $optio
 
     protected function describeRoute(Route $route, array $options = [])
     {
+        $defaults = $route->getDefaults();
+        if (isset($defaults['_controller'])) {
+            $defaults['_controller'] = $this->formatControllerLink(
+                $defaults['_controller'],
+                $this->formatValue($defaults['_controller'])
```

Comment:
should be formatCallable actually

Reply:
I used formatValue to keep the current format but ok, formatCallback is the right way; updated